code/modules/clothing/spacesuits/void/neotheology.dm

```diff
@@ -119,4 +119,43 @@
 		bio = 100,
 		rad = 100
 	)
-	helmet = /obj/item/clothing/head/helmet/space/void/prime
\ No newline at end of file
+	helmet = /obj/item/clothing/head/helmet/space/void/prime
+
+/obj/item/clothing/head/space/void/NTvoid
+	name = "angel voidsuit helmet"
+	desc = "A voidsuit helmet designed by the church with a most holy mix of biomatter and inorganic matter."
+	icon_state = "ntvoidhelmet"
+	item_state = "ntvoidhelmet"
+	action_button_name = "Toggle Helmet Light"
+	flags_inv = BLOCKHAIR
+	armor = list(
+		melee = 40,
+		bullet = 30,
+		energy = 30,
+		bomb = 30,
+		bio = 100,
+		rad = 50
+	)
+	siemens_coefficient = 0.35
+	light_overlay = "helmet_light"
+
+/obj/item/clothing/suit/space/void/NTvoid
+	name = "angel voidsuit"
+	desc = "A voidsuit designed by the church with a most holy mix of biomatter and inorganic matter."
+	icon_state = "ntvoid"
+	item_state = "ntvoid"
+	matter = list(MATERIAL_PLASTEEL = 8, MATERIAL_STEEL = 10, MATERIAL_BIOMATTER = 29)
+	slowdown = 0.3
```

Comment:
```suggestion
	slowdown = 0.15
```
this is barely better then normal ones lets not punsh people for like 10 more melee